spec/features/managing_step_by_step_pages_spec.rb

```diff
@@ -63,11 +64,11 @@
   end
 
   scenario "User cannot publish a page if it's not 2i approved" do
-    given_there_is_a_step_by_step_page_with_a_link_report
+    given_there_is_a_step_by_step_that_has_been_claimed_for_2i
     when_I_view_the_step_by_step_page
     then_there_should_be_no_publish_button
-    # to be updated once Claim for review is done
-    # and_I_should_see_an_inset_prompt
+    and_I_should_see_an_inset_prompt
```

Comment:
I'd suggest adding a `and_there_should_be_no_schedule_button` step here, so that it matches with `Reviever cannot publish or schedule a page if it's not 2i approved` and then `User cannot schedule a page if it's not 2i approved` can be deleted (since lots of it duplicates this test).